sql/donut.sql

```diff
@@ -0,0 +1,48 @@
+DROP TABLE IF EXISTS users;
+DROP TABLE IF EXISTS members;
+
+CREATE TABLE members (
+    user_id int(11) NOT NULL AUTO_INCREMENT, -- Primary Key
+    uid char(7) NOT NULL,  -- Unique
+    last_name varchar(255) NOT NULL, 
+    first_name varchar(255) NOT NULL,
+    middle_name varchar(255), 
+    email varchar(255) NOT NULL,
+    phone varchar(64) DEFAULT NULL,
+    gender tinyint(2) DEFAULT NULL, -- Numerical code that will be stored in code
+    birthday date DEFAULT NULL,
+    entry_year year(4) DEFAULT NULL,
+    graduation_year year(4) DEFAULT NULL, -- Question: does this 1-to-1 denote year of school (frosh, soph)
+    msc smallint(6) DEFAULT NULL,
+    building varchar(32) DEFAULT NULL,
+    room_num smallint(6) DEFAULT NULL,
+    is_abroad bool DEFAULT 0,
+    major varchar(16) DEFAULT NULL,
+    major2 varchar(16) DEFAULT NULL,
```

Comment:
yeah we currently populate majors by running the "populate student registry" script, and if it crashes because of an undefined major then we add it manually. This is pretty infrequent so it's not a huge burden, but it would be slightly better if the script said "Do you want to add X?".